bin/githubchart

```diff
@@ -5,6 +5,7 @@ require 'optparse'
 
 # rubocop:disable Metrics/BlockLength
 options = {}
+draw_method = :svg
```

Comment:
We probably would want to default this either in options = {} above, or in the GithubChart class's init, and override by adding to option hash